include/sys/spa_impl.h

```diff
@@ -246,6 +246,7 @@ struct spa {
 	metaslab_class_t *spa_log_class;	/* intent log data class */
 	metaslab_class_t *spa_embedded_log_class; /* log on normal vdevs */
 	metaslab_class_t *spa_special_class;	/* special allocation class */
+	metaslab_class_t *spa_special_embedded_log_class; /* log on special */
 	metaslab_class_t *spa_dedup_class;	/* dedup allocation class */
```

Comment:
Future cleanup opportunity: there's possibly enough of these now to make them an array indexed on an enum, and then we don't have to repeat ourselves over and over when doing something to all of them (init/fini, stats, etc).